tools/integration_tests/write_large_files/write_large_files_test.go

```diff
@@ -16,23 +16,50 @@
 package write_large_files
 
 import (
+	"fmt"
 	"log"
 	"os"
 	"testing"
 
 	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/mounting/static_mounting"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/operations"
 	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
 )
 
+const (
+	TmpDir = "/tmp"
+	OneMiB = 1024 * 1024
+)
+
+func compareFileFromGCSBucketAndMntDir(gcsFile, mntDirFile, localFilePathToDownloadGcsFile string) (err error) {
+	err = operations.DownloadGcsObject(gcsFile, localFilePathToDownloadGcsFile)
+	if err != nil {
+		err = fmt.Errorf("Error in downloading object:%w", err)
+		return
+	}
+
+	// Remove file after testing.
+	defer operations.RemoveFile(localFilePathToDownloadGcsFile)
+
+	// DiffFiles loads the entire files into memory. These are both 500 MiB files, hence would have a 1 GiB
+	// requirement just for this step
+	diff, err := operations.DiffFiles(mntDirFile, localFilePathToDownloadGcsFile)
```

Comment:
Ideally, we should compare chunk by chunk. 1 GiB RAM seems a little higher for RAM.

Reply:
We can created another method, which takes a chunk size to compare. We can keep chunk-size as 16 MiB by default.

Reply:
We need to anyways downloading 500MB file from bucket to compare the content. Other 500MB file will be in mntDir.
In this DiffFile function we are just comparing two files. How chunk size will make any. difference?

Reply:
As discussed I'll raise bug for this.